.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
+3.0.2
```

Comment:
I find it funny how many places we don't use that file in 😂 

Should we just axe it?

Reply:
rbenv, which i use, uses this file. i think it's super useful personally